docs/playbooks/index.md

```diff
@@ -20,8 +20,9 @@ iterations and will generally have a start and end date.
 [Live](){ .md-button .md-button--disabled }
 [Retirement](){ .md-button .md-button--disabled }
 
-## How to get started as a Developer
+## [How to get started as a Developer](./how-to-get-started/index.md)
 
+This Playbook covers: 
 * What are the CPS Development Principles
```

Comment:
You'll want a new line preceding the first bullet point here and on line 35. If the line before a bullet point is not empty, you'll end up with the following rendering:

This Playbook covers: * What are the CPS Development Principles

Instead of:

This Playbook covers: 

* What are the CPS Development Principles

Reply:
D'oh!

Reply:
done